src/jats/__tests__/__snapshots__/jats-importer.test.ts.snap

```diff
@@ -17912,7 +17912,7 @@ Object {
             },
             Object {
               "attrs": Object {
-                "category": "MPSectionCategory:materials-method",
+                "category": "MPSectionCategory:subsection",
```

Comment:
Are you sure this is the correct behavior?

Reply:
I have tested the changes, and everything is working just fine. Regarding the previous case, it is indeed correct since it’s a subsection.